js/actions/index.ts

```diff
@@ -198,17 +198,42 @@ function watchFirestoreFeedbackSettings(db: firebase.firestore.Firestore, rawPor
   });
 }
 
+interface IStringMap {
```

Comment:
You would get better type checking on the callers if you typed this code like this:

```
interface FeedbackKeys {
    platform_id: string;
    platform_user_id: string;
    resource_link_id: string;
    context_id: string;
}

const feedbackKeys: FeedbackKeys = {
    platform_id: "platformId",
    platform_user_id: "platformStudentId",
    resource_link_id: "resourceLinkId",
    context_id: "contextId"
};

export function correctKey(keyName: keyof FeedbackKeys, receiveMsg: string) { 
 ```

then correctKey("foo", "bar") wouldn't compile.

Reply:
Interesting, I didn't know about `keyof`. I think in this case using an array of possible keyNames would make more sense and then have feedbackKeys restrict its keys with that. But since this is intended as an internal method and will hopefully all go away, it doesn't seem worth it.

An tangential thing to note about this project is that when I run `npm run lint` it doesn't check the types as strictly as when I run `npm test`. I guess jest must be using a different typescript configuration than the lint command is using.